### ISSUE TITLE
Consideration for JWK <-> PEM formats

### ISSUE THREAD

**Issue by nicktacular** [latchset/jose#39]
I was curious if you guys considered having a conversion build in for JWK <-> PEM formats? Perhaps even extension based so that if `-o whatever.pem` ended in PEM then you would use that format.

**Comment by npmccallum** [comment]
Yes, we have. I even have a (very stale) branch for working on this. But getting the core API right and usable CLI was a higher priority. Patches are welcome!

**Comment by asiplas** [comment]
#62 is a start—only useful for signing at the moment and needs some testing and review.

**Comment by asiplas** [comment]
Cursory testing—verification via https://jwt.io/ seems OK.
